microblaze/anal/anal_microblaze_gnu.c

```diff
@@ -9,638 +9,630 @@
 #include "microblaze-opc.h"
 #include "dis-asm.h"
 
-#define get_int_field_imm(instr)   ((instr & IMM_MASK) >> IMM_LOW)
-#define get_int_field_r1(instr)    ((instr & RA_MASK) >> RA_LOW)
-#define get_int_field_r2(instr)    ((instr & RB_MASK) >> RB_LOW)
+#define UT32_16U UT16_MAX << 16
+#define UT32_16L UT16_MAX
 
-#define get_field_rd(instr)        get_field (instr, RD_MASK, RD_LOW)
-#define get_field_r1(instr)        get_field (instr, RA_MASK, RA_LOW)
-#define get_field_r2(instr)        get_field (instr, RB_MASK, RB_LOW)
-#define get_field_imm(instr)       get_imm   (ctx, instr)
+#define get_int_field_imm(instr) ((instr & IMM_MASK) >> IMM_LOW)
+#define get_int_field_r1(instr) ((instr & RA_MASK) >> RA_LOW)
+#define get_int_field_r2(instr) ((instr & RB_MASK) >> RB_LOW)
+
+#define get_field_rd(instr) get_field(instr, RD_MASK, RD_LOW)
+#define get_field_r1(instr) get_field(instr, RA_MASK, RA_LOW)
+#define get_field_r2(instr) get_field(instr, RB_MASK, RB_LOW)
+#define get_field_imm(instr) get_imm(ctx, instr)
 
 struct mb_anal_ctx {
-	int immval;
-	bool immfound;
-	bool immfound_addr;
-	RAnal *anal;
-	RAnalOp *op;
+  ut16 immval;
```

Comment:
Is this a tab? Because from phone it looks like 2 spaces

Reply:
It is two space but that's the tool **sys/clang-format-diff.py** from [radare2](https://github.com/radare/radare2/blob/master/CONTRIBUTING.md) which changed it. Do I have to revert it ?